MapboxNavigation/NavigationView.swift

```diff
@@ -75,12 +75,11 @@ open class NavigationView: UIView {
     }
     
     lazy var mapView: NavigationMapView = {
-        let map: NavigationMapView = .forAutoLayout()
+        let map: NavigationMapView = .forAutoLayout(frame: self.bounds)
```

Comment:
Initializing an `MGLMapView` with a zero rect will expand to 64x64. This can cause weird behavior and slow 
 autolayout animations. @JThramer 👀